### DISCUSSION TITLE
invitation to slack

### DISCUSSION THREAD

**Discussion by aziizz** [nasa-jpl/open-source-rover#286] [Q&A] [answered]
Hi ! can you send me an invitation to slack please ?
Thanks !

**Comment by ericjunkins** [comment, marked as answer]
Hi! I updated the README of the main page to include a link to join the slack discussion, it can be seen here:

https://github.com/nasa-jpl/open-source-rover#communication

Here is also the direct link: 

https://join.slack.com/t/opensourcerover/shared_invite/zt-mfe0viua-FhcYsNb3S9vUra99uF3IBw
